src/script/campaign/mod/vco-listeners.lua

```diff
@@ -256,6 +262,22 @@ local function add_listeners()
 		true
 	);
 
+	vco:log("- Skaven listeners");
+	core:add_listener(
+		"vco_skv_moulder_upgrade_count_check",
+		"UnitTrained",
+		function(context)
+			vco:log("inside mdr listener conditional check")
+			local faction = context:unit():faction()
+			return faction:name() == "wh2_main_skv_clan_moulder" and faction:is_human()
```

Comment:
S: Now that I think about it, checking if the faction is human before checking the name is more efficient. We should actually modify it in all the listeners. 

Also please add `;` at the end of the lines. I know this is lua and it is not required, but it makes papa happy.